book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, person)
+    add_rental(rental)
+  end
```

Comment:
- [ ] [Optional] It would be better if you send the book itself as a parameter to create the new rental this way 
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
end
```

Reply:
thanks for the review